src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2754,6 +2844,15 @@ public ConstructorEdit(INamedTypeSymbol oldType)
             return semanticEdits.ToImmutable();
         }
 
+        private void ReportRecordSynthesizedMembersChanged(Compilation compilation, INamedTypeSymbol recordType, ArrayBuilder<SemanticEditInfo> semanticEdits)
```

Comment:
>Report [](start = 21, length = 6)

I'd call this `AddEditsForSynthesizedRecordMembers`. "Report" is usually used for reporting diagnostics.